### PR TITLE
issue-214: Update to Jakarta EE8 APIs

### LINKED ISSUE #214: Update to Jakarta EE8 APIs for MP 4.0 in Jun 2020

### PR DESCRIPTION
**Pull request by xstefank**
resolves #214

### CODE REVIEW

**Comment by xstefank** [comment]
@pgunapal can you also take a look, please?

File: api/pom.xml [hunk 1]
```diff
@@ -28,8 +28,8 @@
 
     <dependencies>
         <dependency>
-            <groupId>javax.enterprise</groupId>
-            <artifactId>cdi-api</artifactId>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-api</artifactId>
```
Comment (pgunapal) [inline api/pom.xml:32]: Should we mark all the dependencies as provided in the pom.xml for API and TCKs in this PR? Unless, we want another PR to address that for Issue #220

File: pom.xml [hunk 2]
```diff
@@ -119,20 +119,20 @@
...
+                <groupId>jakarta.enterprise</groupId>
+                <artifactId>jakarta.enterprise.cdi-api</artifactId>
                 <version>${cdi-api.version}</version>
                 <exclusions>
                     <exclusion>
-                        <groupId>javax.el</groupId>
-                        <artifactId>javax.el-api</artifactId>
+                        <groupId>jakarta.el</groupId>
+                        <artifactId>jakarta.el-api</artifactId>
                     </exclusion>
```
Comment (pgunapal) [inline pom.xml:129]: Do we need an EJB exclusion here, as well?

File: api/pom.xml (hunk 1 above)
Comment (xstefank) [reply api/pom.xml:32]: I'll do that in different issue, so we have issue resolved with separate PR for consistency.

File: pom.xml (hunk 2 above)
Comment (xstefank) [reply pom.xml:129]: I'll do that in different issue, so we have issue resolved with separate PR for consistency.

**Comment by xstefank** [comment]
@pgunapal I take +1 means that this PR is ok with you? :) I will merge it then.

**Comment by pgunapal** [comment]
@xstefank I was thinking of it as more of a "thumbs up", than a +1 for that case :).

**Comment by antoinesd** [comment]
LGTM

**Pull request merged**